packages/xo-web/src/common/xo/index.js

```diff
@@ -495,12 +495,11 @@ subscribeCheckSrCurrentState.forceRefresh = pool => {
 const missingPatchesByHost = {}
 export const subscribeHostMissingPatches = (host, cb) => {
   const hostId = resolveId(host)
-
-  if (missingPatchesByHost[hostId] == null) {
-    missingPatchesByHost[hostId] = createSubscription(() =>
-      getHostMissingPatches(host)
-    )
-  }
+  host.power_state !== 'Running'
+    ? (missingPatchesByHost[hostId] = () => [])
+    : (missingPatchesByHost[hostId] = createSubscription(() =>
+        getHostMissingPatches(host)
```

Comment:
The case of the host stop is not handled in `xo-app/menu/index`. To avoid duplicate the logic in multiple UIs, you can do it her.

To get the host object you can use `getObject(store.getState(), hostId)`. Then, if the host is not running you can unsubscribe.

Reply:
The above comment isn't good because we create a new subscription on each host state change.